src/lang/po/Prusa-Firmware-Buddy.pot

```diff
@@ -1065,7 +1019,18 @@ msgid ""
 " - Selected filament"
 msgstr ""
 
-#: src/gui/screen_menu_lan_settings.cpp:296
+#: src/gui/wizard/screen_wizard.cpp:304
+msgid ""
+"The status bar is at the bottom of the screen. It contains information "
```

Comment:
duplicit

Reply:
When the #if 0 code goes away, the duplicity will be removed. No change for now.